### ISSUE TITLE
curlimages/curl:8.1.0 fails on a simple GET

### ISSUE THREAD

**Issue by mottolini** [curl/curl-docker#82]
In a gitlab pipeline I'm using `curlimages/curl` to execute simple curl commands like:

    curl -H "$HEADER_ACCEPT" -u "$CREDENTIALS" "https://${DOCKER_REGISTRY}/v2/${DOCKER_REPO}/manifests/${CI_COMMIT_SHA}"

It worked well with `latest` tag until the release of 8.1.0. The error now is:

    * Closing connection -1
    curl: (3) URL using bad/illegal format or missing URL

Reverting back to `7.88.1` solved the problem

**Comment by bagder** [comment]
Is this possibly due to https://github.com/curl/curl/issues/11129 ?

**Comment by xquery** [comment]
yes probably

**Comment by mottolini** [comment]
Right! The registry is ECR and the url starts with some digits. Feel free to close this issue or leave it open until fixed
